### PR TITLE
Fix/tab widget aria role

### PR DESCRIPTION
**Pull request by MaxBentz**
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been documented
- [x] Changes have been tested by the author
- [x] Changes have been tested by the reviewer

@plentymarkets/ceres-io

fixing the following error by placing the role-attribute "tab" to the direct children: List-Element (\<li>)

grafik

### CODE REVIEW

**Comment by MaxBentz** [comment]
Is it necessary to run a new build when commit a change in vue components?

File: resources/js/src/app/components/common/TabList.vue [hunk 1]
```diff
@@ -39,6 +38,7 @@ const TabNavItem = {
         return createElement(
             "li",
             {
+                role: "tab",
```
Comment (stentrop) [inline resources/js/src/app/components/common/TabList.vue]:
```suggestion
                attrs: {role: "tab"},
```

Comment (stentrop) [reply resources/js/src/app/components/common/TabList.vue]: @cerwantes 

i think this is the correct syntax.

**Review by stentrop — CHANGES_REQUESTED** [review]

**Comment by stentrop** [comment]
> Is it necessary to run a new build when commit a change in vue components?

If you use an own fork of ceres, then it is necessary. For our stable branch it is not necessary, there is an automatic build on merge.

**Review by stentrop — APPROVED** [review]

**Pull request merged**